DSCResources/MSFT_xVHD/MSFT_xVHD.psm1

```diff
@@ -198,7 +229,18 @@ function Test-TargetResource
     {
        Throw "Either specify ParentPath or MaximumSizeBytes property." 
     }
-            
+
+    # Check the module combinations make sense
```

Comment:
Could you remove this comment or move it up (so the earlier validation is included under the comment as well) and change the wording so that it says something like, Validating Parameters